src/Symfony/Component/Cache/Traits/MemcachedTrait.php

```diff
@@ -206,14 +206,7 @@ protected function doSave(array $values, $lifetime)
      */
     protected function doFetch(array $ids)
     {
-        $unserializeCallbackHandler = ini_set('unserialize_callback_func', __CLASS__.'::handleUnserializeCallback');
-        try {
-            return $this->checkResultCode($this->getClient()->getMulti($ids));
```

Comment:
serialization is also handled by the extension itself, so that the Memcached should be provided an IdentitySerializer and this could should be kept as is, isn't it?

Reply:
I did not add SerilaizerTrait to Caches that handle serialization by itselves: Memcached, Apcu, Doctrine. It looks like adding IdentitySerilizer to it is good idea.

Reply:
it would allow using a compressing serializer if needed, so would be great yes